feat: add Language contribution guide in README (#120)
bndw/wifi-card: README.md

make fmt.write
```

## Language Contribution Guide

We would love for you to contribute to different languages and help make it even better than it is today! As a language contributor, here are some steps we would like you to follow:

1. Add a translation to `./src/i18n.js`

> 2. Add the language option to `./src/app.js`. Use the format e.g. `French (Français)`

Reviewer: ```suggestion
2. Add the language option to `./src/App.js`. Use the format e.g. `French (Français)`
```

make fmt.write
```

## Language Contribution Guide

We would love for you to contribute to different languages and help make it even better than it is today! As a language contributor, here are some steps we would like you to follow:

> 1. Add a translation to `./src/i18n.js`

Reviewer 2: Should we make these hyperlinks?
```suggestion
1. Add a translation to [`./src/i18n.js`](./src/i18n.js)
```

Author: looks good

> 2. Add the language option to `./src/App.js`. Use the format e.g. `French (Français)`

Reviewer 2: As per https://github.com/bndw/wifi-card/issues/80#issuecomment-887598899, looks like we should do `latinName - nativeName` instead of `latinName (nativeName)` to support RTL languages.

Let's wait until https://github.com/bndw/wifi-card/pull/149 shakes out to determine.

> 2. Add the language option to `./src/App.js`. Use the format e.g. `French (Français)`

Reviewer 2: ```suggestion
2. Add the language option to [`./src/App.js`](./src/App.js) in the format `latinName - nativeName`, e.g. `French - Français`
```
